dictionaries/condensed-strokes.json

```diff
@@ -1,6 +1,10 @@
 {
+"TRA*FL/HRER": "traveller",
```

Comment:
@paulfioravanti could this be `TRAFL/HRER`?

Reply:
`TRAFL/HRER` is already in the dictionaries as "traveler" with one "l". In order to get English spelling, it needed the asterisk.